uvicorn/loops/uvloop.py

```diff
@@ -4,6 +4,5 @@
 
 
 def uvloop_setup():
-    asyncio.get_event_loop().close()
     asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
     return asyncio.get_event_loop()
```

Comment:
I guess we don't actually need to return the loop from our setup functions anymore, since we don't use the return value now.